src/Liquid.Activation/Scheduler/Scheduler.cs

```
@@ -0,0 +1,84 @@
+﻿using Liquid.Base.Interfaces;
+using Liquid.Runtime.Telemetry;
+using System;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Liquid.Activation
+{
+    /// <summary>
+    /// Implementation of the communication component  of Scheduler 
+    /// </summary>
+    public class Scheduler : LightScheduler, IWorkBenchHealthCheck
+    {
+        /// <summary>
+        /// Implementation of the start process Scheduler. It must be called  parent before start processes.
+        /// </summary>
+        public override void Initialize()
+        {
+            base.Initialize();
+            StartWorkerAsync();
+        }
+
+        protected async Task StartWorkerAsync()
+        {
+            try
+            {
+                foreach (var scheduling in _scheduler)
+                {
+                    MethodInfo method = GetMethod(scheduling);
+                    string name = scheduling.Value.Name;
+                    CancellationToken _cancelToken = new CancellationToken(); 
+                    await StartAsync(_cancelToken, name, method);
+                }
+            }
+            catch (Exception exception)
+            {
+                Exception moreInfo = new Exception($"Error setting up queue consumption from scheduler. See inner exception for details. Message={exception.Message}", exception);
```

It seems that we fail silently here. Should be throw the new exception?